OpenEdXMobile/src/main/java/org/edx/mobile/user/DeleteAccountImageTask.java

```diff
@@ -1,43 +1,59 @@
 package org.edx.mobile.user;
 
 import android.content.Context;
-import androidx.annotation.NonNull;
 
-import com.google.inject.Inject;
+import androidx.annotation.NonNull;
 
+import org.edx.mobile.core.EdxDefaultModule;
 import org.edx.mobile.event.ProfilePhotoUpdatedEvent;
 import org.edx.mobile.module.prefs.LoginPrefs;
 import org.edx.mobile.task.Task;
 
+import java.io.IOException;
+
+import dagger.hilt.android.EntryPointAccessors;
 import de.greenrobot.event.EventBus;
 
 public class DeleteAccountImageTask extends
         Task<Void> {
 
-    @Inject
-    private UserService userService;
-
-    @Inject
-    private LoginPrefs loginPrefs;
+    UserService userService;
+    LoginPrefs loginPrefs;
 
     @NonNull
     private final String username;
 
     public DeleteAccountImageTask(@NonNull Context context, @NonNull String username) {
         super(context);
         this.username = username;
+        EdxDefaultModule.ProviderEntryPoint provider = EntryPointAccessors.fromApplication(context.getApplicationContext(), EdxDefaultModule.ProviderEntryPoint.class);
```

Comment:
Can we add `@ApplicationContext` in method definition instead of using `context.getApplicationContext()`?